_members/eric-hurwitz

```diff
@@ -0,0 +1,22 @@
+---
+name: Eric Hurwitz
+image: 
```

Comment:
@ehurwitz - please add the link to your headshot in row three.
`image: images/team/eric-hurwitz.jpg`